README.md

```diff
@@ -82,7 +82,8 @@ If you want to specify a specific commit as the source of the package append the
 
 ## Conventions
 
-We follow the [IAI Python Style Guide](https://github.com/iai-group/styleguide/tree/main/python).
+We follow the [IAI Python Style Guide](https://github.com/iai-group/styleguide/tree/main/python), except that we assume Python 3.9.
```

Comment:
Those guidelines will also be updated to Python 3.9 (https://github.com/iai-group/guidelines/issues/45), so you can skip that part. But it'd be good to have a separate Prerequisites section where we state the Python version and any other dependencies.

Reply:
Done.